models/articles.models.js

```diff
@@ -10,3 +10,19 @@ exports.getArticlesById = (article_id) => {
       return result.rows[0];
     });
 };
+
+exports.getAllArticles = () => {
+  return db
+    .query(
+      `
+    SELECT articles.article_id, articles.title, articles.author, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id)::INTEGER AS comment_count
```

Comment:
Great use of the count sub function, really like that you've cast this as an integer aswell here!